### DISCUSSION TITLE
[Feature] Family Relationship Management and Visualization

### DISCUSSION THREAD

**Discussion by himself85** [immich-app/immich#14978] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Currently, Immich allows face recognition for individuals but lacks the ability to establish and visualize family relationships between recognized people in photos.
Proposed Solution
A comprehensive family relationship management system that enables:
Ability to define relationships between recognized individuals
Parent-child relationship assignments
Visualization of family connections in a tree view
Interactive family tree display with photo thumbnails
Flexible relationship types (biological, step, adopted)

### Platform

- [ ] Server
- [ ] Web
- [ ] Mobile

**Comment by mmomjian** [comment]
Thank you, this is however outside the scope of what immich plans to offer. Perhaps a good choice for an external tool or project of some kind.

**Comment by SelfHostedEnv** [comment]
Great feature, even though I know its outside of Immich description.
I came here to post about the feature and someone already created discussion about it.
Does Immich support plugin or extension system?

I think most of people who use Immich as an media solution for family would love to able to visualize relationship as well, especially when family getting larger
